main.py

```diff
@@ -1,17 +1,24 @@
+import logging
+
 import uvicorn
 from fastapi import FastAPI
 from app.database import engine, Base
 from app.routers import endpoints
+from process_csv import run_script
+from app.utils.logger_config import setup_logger
 
 Base.metadata.create_all(bind=engine)
 # FastAPI app instance
 app = FastAPI()
 
+app_logger = setup_logger('app_logger', logging.INFO)
```

Comment:
nice, but why don't you call the env var here? I think that dependency should be everywhere, since its either `DEV` env, either `PROD` env.